code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -123,7 +123,8 @@
 		new /obj/item/weapon/storage/belt/security(src)
 		new /obj/item/device/flash(src)
 		new /obj/item/weapon/melee/baton/loaded(src)
-		new /obj/item/weapon/gun/energy/gun(src)
+		new /obj/item/weapon/storage/lockbox/lawgiver
```

Comment:
You have to include the (src) argument to tell the item where to spawn. Did this work on its own?